pkg/cluster/internal/create/actions/createworker/createworker.go

```diff
@@ -129,34 +155,49 @@ func (a *action) Execute(ctx *actions.ActionContext) error {
 
 	ctx.Status.End(true) // End Generating worker cluster manifests
 
-	ctx.Status.Start("Generating secrets file 📝🗝️")
-	defer ctx.Status.End(false)
+	_, err = os.Stat("./secrets.yml")
```

Comment:
since we reference secrets.yml multiple times, could be make it a constant?

Reply:
Done